src/components/TextField.vue

```diff
@@ -29,6 +30,7 @@
       :for="id"
     >
       {{ label }}
+      <span v-if="required">(required)</span>
```

Comment:
```suggestion
      <span v-if="required" class="text-field__required-text">(required)</span>
```

Reply:
ok got it thanks Guy, im going to look for a crash course 101 on BEM, need to nail this down!